### PR TITLE
allow multiple delete

### PR DESCRIPTION
**Pull request by JacquesBorsenberger**

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/NetworkModificationController.java [hunk 1]
```diff
@@ -82,11 +83,12 @@ public NetworkModificationController(NetworkModificationService networkModificat
         return ResponseEntity.ok().body(networkModificationService.moveModifications(groupUuid, before, modificationsToMove));
     }
 
-    @DeleteMapping(value = "/groups/{groupUuid}/modifications/{modificationUuid}")
-    @Operation(summary = "Delete a modification from a group")
-    @ApiResponse(responseCode = "200", description = "Modification deleted")
-    public ResponseEntity<Mono<Void>> deleteModifications(@Parameter(description = "Group UUID") @PathVariable("groupUuid") UUID groupUuid, @Parameter(description = "Modification UUID") @PathVariable("modificationUuid") UUID modificationUuid) {
-        return ResponseEntity.ok().body(networkModificationService.deleteModification(groupUuid, modificationUuid));
+    @DeleteMapping(value = "/groups/{groupUuid}/modifications")
+    @Operation(summary = "Delete a modifications from a group")
```
[the remark this answers is not in the document]
Comment (JacquesBorsenberger) [reply src/main/java/org/gridsuite/modification/server/NetworkModificationController.java]: fixed

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java [hunk 2]
```diff
@@ -1404,9 +1404,9 @@ private void sendCancelBuildMessage(Message<String> message) {
         publisher.send("publishCancelBuild-out-0", message);
     }
 
-    public Mono<Void> deleteModification(UUID groupUuid, UUID modificationUuid) {
+    public Mono<Void> deleteModifications(UUID groupUuid, Set<UUID> modificationUuid) {
```
Comment (niconoir) [inline src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java:1407]: Typo: modification**s**Uuid**s**

**Review by niconoir — COMMENTED** [review]
Maybe modify testDeleteModification from ModificationControllerTest to test deletion of more than one modification.

File: src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java (hunk 2 above)
Comment (JacquesBorsenberger) [reply src/main/java/org/gridsuite/modification/server/service/NetworkModificationService.java]: fixed

**Review by FranckLecuyer — APPROVED** [review]

**Review by niconoir — APPROVED** [review]

**Pull request merged**